features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml

```diff
@@ -1179,6 +1179,12 @@
     <Resource context="(.*)/o/api/server/v1/organizations/check-handle" secured="true" http-method="POST">
         <Scopes>internal_org_organization_view</Scopes>
     </Resource>
+    <Resource context="(.*)/o/api/server/v1/organizations/self" secured="true" http-method="GET">
+        <Scopes>internal_org_organization_view</Scopes>
+    </Resource>
+    <Resource context="(.*)/o/api/server/v1/organizations/self" secured="true" http-method="PATCH">
+        <Scopes>internal_org_organization_update</Scopes>
+    </Resource>
```

Comment:
Ideally these two should go before the GET and PATCH of (.*)/o/api/server/v1/organizations/(.*)

Since we have used the same scope as regex path having here doesn't make any difference, but level move before them
Fix in other places as well